.gitignore

```diff
@@ -14,3 +14,5 @@ opactl_*
 /files
 /.go
 /dist
+
+.env
```

Comment:
Can we keep that file as-is, please? There's nothing inside the repo that's using .env, I believe.